### ISSUE TITLE
Game freezes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#221]
```
What steps will reproduce the problem?
1. Play the game
2. Switch rounds

When i am playing after sometime my game freezes(except the background like the 
snow falling) fire stops moving, players stop moving etc
i see at the top left corner a little pc with moving arrow dots.

It happens after like 5mins of starting the game.

Os: ubuntu 10:10
friends os: debian 6
```

-----
Original issue reported on code.google.com by `lucas.ho...@gmail.com` on 20 Apr 2011 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
little adition: after waiting for some minutes the game unfreezes and all 
events happen at speed X2 until its up to date, then it freezes again.
```

-----
Original comment by `lucas.ho...@gmail.com` on 20 Apr 2011 at 4:56

**Comment by GoogleCodeExporter** [comment]
```
Indicates network lag.

There is some problem with one of your connections.

Try traceroute/ping.

long pings (15 minutes or more, 1 per second) may indicate periodic 
interruptions delays in connectivity due to a bad internet connection.


```

-----
Original comment by `kyberneticist@gmail.com` on 20 Apr 2011 at 5:12
* Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
That is not the case, we have checked our networks and still it happens after 
about 5 minutes, the whole game freezes. We played version 0.9.13 the same day 
as when we updated(which is today) and it still happens
```

-----
Original comment by `lucas.ho...@gmail.com` on 20 Apr 2011 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
Erm. 0.9.13? Why are you not on 0.9.15 - for ubuntu you can get it off of 
PlayDeb.

ReadyTimer is a possibility for bugs, but that was not introduced until six 
months later.

Anyway... If there is in fact some bug you discovered in the network protocol, 
odds are good it has been fixed in the year since 0.9.13 was released, so, not 
going to reopen this until you test with latest version.

Debugging old code leads to nothing productive in the end.

As for you being sure it isn't your network.  0.9.13 was a quite stable 
release, which was why it lasted for so long. There was no pressure to release 
0.9.14.

Are you connecting to the official game server? If so, please verify your 
connection at EACH hop.

Also try a long running ping against the official game server.

Also, consider trying to run a local game server on your LAN - if that is fine, 
it is your network connection, somewhere between the official server and you.

```

-----
Original comment by `kyberneticist@gmail.com` on 20 Apr 2011 at 8:54
